tests/kafkatest/tests/mirror_maker_test.py

```diff
@@ -13,17 +13,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-from ducktape.tests.test import Test
+from ducktape.utils.util import wait_until
 
 from kafkatest.services.zookeeper import ZookeeperService
 from kafkatest.services.kafka import KafkaService
 from kafkatest.services.console_consumer import ConsoleConsumer
 from kafkatest.services.verifiable_producer import VerifiableProducer
 from kafkatest.services.mirror_maker import MirrorMaker
+from kafkatest.tests.produce_consume_validate import ProduceConsumeValidateTest
+
+
+def clean_bounce(test):
+    for i in range(3):
+        test.logger.info("Bringing mirror maker nodes down...")
+        for node in test.mirror_maker.nodes:
+            test.mirror_maker.stop_node(node)
+
+        test.logger.info("Bringing mirror maker nodes back up...")
+        for node in test.mirror_maker.nodes:
+            test.mirror_maker.start_node(node)
+
+
+def hard_bounce(test):
```

Comment:
Couldn't these just be methods of TestMirrorMakerService and then you could just pass `core_test_action=self.hard_bounce`? Seems more intuitive to me to keep it as a method since it only seems to be used with the one class.